map/framework.hpp

```diff
@@ -166,7 +161,7 @@ class Framework : public search::ViewportSearchCallback::Delegate,
 
   unique_ptr<ugc::Api> m_ugcApi;
 
-  unique_ptr<search::Engine> m_searchEngine;
+  unique_ptr<SearchAPI> m_searchAPI;
```

Comment:
nit: We use "Api" instead of "API" everywhere.

Reply:
I know, but API is the right naming, because it's an abbreviation. Therefore I'm insist on the right way, not on the convenient name.